lib/Vector2D.js

```diff
@@ -151,6 +151,8 @@ Vector2D.prototype.divide = function(scalar) {
  */
 Vector2D.prototype.angleBetween = function(that) {
     var cos = this.dot(that) / (this.length() * that.length());
+    if (cos < -1) cos = -1;
+    else if (cos > 1) cos = 1;
```

Comment:
Didn't know that.

BTW, I prefer if/else statements to always use curly braces. Can you change this to following?:

```
if (cos < -1) {
  cos = -1;
}
else if (cos > 1) {
  cos = 1;
}
```

I guess if there are going to be contributors, I should put up a style guide.